acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -80,7 +83,7 @@ export default function init(element) {
   if (preRender) {
     (async () => {
       // TODO: Make dynamic
-      const response = await fetch(DC_GENERATE_CACHE_URL || `https://documentcloud.adobe.com/dc-generate-cache/dc-hosted-1.165.0/${VERB}-${pageLang.toLocaleLowerCase()}.html`);
+      const response = await fetch(DC_GENERATE_CACHE_URL || `https://documentcloud.adobe.com/dc-generate-cache/dc-hosted-${DC_GENERATE_CACHE_VERSION}/${VERB}-${pageLang.toLocaleLowerCase()}.html`);
```

Comment:
this url can be generated from metadata if you choose `content.dc-generate-cache.urls.{$ENV}`

Reply:
We are using https://adobe.sharepoint.com/:x:/r/sites/adobecom/_layouts/15/Doc.aspx?sourcedoc=%7BBC0B7771-604F-4E0F-8319-FA0D6AEEFCBD%7D&file=metadata.xlsx&action=default&mobileredirect=true  to generate metadata, and that file can be changed by authors, so they can update to wanted version . 

We can fetch urls from https://acrobat.adobe.com/dc-hosted/private/info.json for dc  widget version that is set in excel metadata, but not sure is it really necessary, @Blainegunn ?

Reply:
info.json should be the source of the versions as well.  this should be automated somehow.

Reply:
@peller baby steps

Reply:
@Blainegunn @peller yes, that was part of our plan - second step would be to create rundeck job which will generate widget version and update excel sheet based on https://acrobat.adobe.com/dc-hosted/private/info.json

Reply:
Ok, then you may want to substitute the partial urls here instead of just the version number, but that can be a follow up